### PR TITLE
New Difficult Feature

### PR DESCRIPTION
**Pull request by NathanC55**
test

### CODE REVIEW

**Comment by NathanC55** [comment]
im unable to import Words from words file. Its not giving me any error on compile but when I run it JS is not letting me

File: src/components/words.ts [hunk 1]
```diff
@@ -0,0 +1,5761 @@
+const WORDS = [
```
Comment (leviFrosty) [inline src/components/words.ts:1]: Need to `export const` here

Comment (NathanC55) [reply src/components/words.ts:1]: still getting error

File: src/html/main.html [hunk 2]
```diff
@@ -132,5 +147,6 @@
     </main>
     <script src="/dist/components/main.js"></script>
     <script src="/dist/components/statistics.js"></script>
+    <script src="/dist/components/settings.js"></script>
```
Comment (leviFrosty) [inline src/html/main.html:150]: Need to import `words.js` here in order to use JS

**Pull request merged**